tests/test_export.py

```diff
@@ -0,0 +1,131 @@
+import pytest
+import json
+from jsonschema import validate
+
+from tabgenie.utils.export import *
+
+'''-------------------JSON-SCHEMA-----------------------'''
+# https://python-jsonschema.readthedocs.io/en/latest/
+# need schema for testing tabs exporting into json
+with open('tests/schemes/schema_without_prop.json') as f:
+    schema_without_prop = json.load(f)
+
+with open('tests/schemes/schema_with_prop.json') as f:
+    schema_with_prop = json.load(f)
+'''-----------------------------------------------------'''
+
+
+def test_export_json(prepare_tests):
+    tab, split, len_tab, cls = prepare_tests
+    failed_nums = []
+    for nmb in range(len_tab):
+        try:
+            table = cls.prepare_table(cls.data[split][nmb])
```

Comment:
and here, if you do the loop like I suggested, you do `table = cls.prepare_table(raw_table)`